oar/core/util.py

```diff
@@ -36,7 +36,10 @@ def init_logging(log_level=logging.INFO):
         if "requests" in k or "urllib3" in k or "gssapi" in k:
             logger = logging.getLogger(k)
             logger.setLevel(logging.WARNING)
-
+    for k in loggers.keys():
```

Comment:
@wewang58 I think you can add another if block in original loop, new for loop is not needed here.

Reply:
@rioliu-rh  Updated, please check again, thanks